articles/api-auth/tutorials/using-resource-owner-password-from-server-side.md

```diff
@@ -20,7 +20,7 @@ Server-side applications can use the [Resource Owner Password Grant](/api-auth/g
 
 ## Prerequisites
 
-Before you continue, make sure to have [brute force protection](/anomaly-detection/guides/enable-disable-brute-force-protection) enabled from your dashboard.
+Before you continue, make sure to have [brute force protection](/anomaly-detection/guides/set-anomaly-detection-preferences) enabled from your Dashboard.
```

Comment:
Do we have an anchor for the Brute Force section of that page?

Reply:
No